input/pagecontent/index.md

```diff
@@ -1007,7 +1017,7 @@ Mixins are defined by using the keywords `Mixin`, `Parent`, and `Description`.
   * ^jurisdiction.coding = COUNTRY#US "United States of America"
 ```
 
-#### Defining Invariants
+#### 🚧 Defining Invariants
```

Comment:
Again here we have the 🚧, but on line 787 for Invariants you have the 🚫. I'm thinking that maybe this is because you have the 🚫 on xpath, but overall it feels a little inconsistent.

Reply:
fixed.